app/com/linkedin/drelephant/util/Utils.java

```diff
@@ -102,7 +102,7 @@ public static Document loadXMLDoc(String filePath) {
     Map<String, String> options = new HashMap<String, String>();
     String[] tokens = str.trim().split("\\s");
     for (String token : tokens) {
-      if (token.isEmpty()) {
+      if (token.isEmpty() || token.startsWith("-X")) {
```

Comment:
we have options like `-XX:+UseCompressedOops -XX:MaxPermSize=512m` that was blowing Java option parsing away